change/react-native-windows-0eabcbbe-3124-42fe-b30b-356332950c73.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Something seems wrong with this, file.. Maybe it requires regeneration...

Reply:
0.69 hasn't released yet, so this should only be a pre-release bump.